subsys/bluetooth/host_extensions/Kconfig

```diff
@@ -58,6 +57,11 @@ config BT_RADIO_NOTIFICATION_CONN_CB_EGU_INST_EGU5
 	depends on $(dt_nodelabel_has_compat,egu5,$(DT_COMPAT_NORDIC_NRF_EGU))
 	select NRFX_EGU5
 
+config BT_RADIO_NOTIFICATION_CONN_CB_EGU_INST_EGU10
+	bool "Use EGU10"
+	depends on $(dt_nodelabel_has_compat,egu10,$(DT_COMPAT_NORDIC_NRF_EGU))
+	select NRFX_EGU10
+
```

Comment:
Should the egu selection be moved to devicetree instead so that we dont have to list all of the instances both here and in the c file? It would also be no work to add support for potential new instances in new hardware.

If the application could do something like this it would be quite nice I think.
```dts
/ {
  chosen {
    nordic,sdc-radio-notification = &egu10;
  };
};
```

Reply:
Yes, that would scale better. Should we do it as part of this PR or a followup?
Note: We may at some point consider to remove the usage of EGU in this module completely and make the trigger timestamp-based instead.

Reply:
Dont have to do it now, I am fine with just adding the new case for 54L

Reply:
Should EGU20 be added for 54L?

Reply:
Yes, we can add it. I will anyways move this driver to be DTS based, and then this will be available automatically. Are you fine with leaving it out for now?

Reply:
Fine with leaving it